src/elements/list-box.svelte

```diff
@@ -0,0 +1,159 @@
+  };
+  
+  options = newOptions;
+};
+
+afterUpdate(addNewData);
+addStyles();
+
+const handleOptionClick = (option: ListBoxOption, side: ListBoxSide) => {
+  if (isDisabled) return;
+
+  option.selected = !option.selected;
+  options = { ...options };
+
+  dispatch('option-click', {
+    target: { ...option, side },
+  });
+};
+
+const handleMoveClick = (target: ListBoxSide) => {
+  if (isDisabled) return;
+
+  const source = target === LEFT ? RIGHT : LEFT;
+
+  const newOptions: Options = {
+    left: [],
+    right: [],
+  };
+  
+  for (const option of options[source]) {
+    if (option.selected) {
+      newOptions[target].push({ ...option, selected: false });
+    } else {
+      newOptions[source].push(option);
+    }
+  }
+
+  options[source] = newOptions[source];
+  options[target] = [...options[target], ...newOptions[target]];
+  options = { ...options };
+
+  // copy object to avoid any potential pass-by-reference issues 
+  dispatch('move', {  
+    options: JSON.parse(JSON.stringify(options)) as Options,
+  });
+};
+
+</script>
+<div
+  class={cx('w-full grid grid-cols-[1fr_32px_1fr] gap-8 items-center p-2', { '!text-black/50': isDisabled })}
+>
+  <div class="w-full flex flex-col gap-2 self-stretch" style={`height: ${height};`}>
+    <span class="text-xs">{ leftlabel }</span>
+    <div class="border border-[#D7D7D] grow px-4 py-3 bg-white flex flex-col overflow-auto">
+      {#each options.left as option}
+        <button
+          class={cx('flex items-start', { 'bg-[#E2F1FD]': option.selected })}
+          on:click={() => handleOptionClick(option, LEFT)}
+        >        
+          <span class="text-sm px-2 py-0.5">{ option.value }</span>
+        </button>
+      {/each}
+    </div>
+  </div>
+  <div class="flex flex-col gap-4">
+    <button
+      class={cx('rotate-90 border border-black h-8 w-8 flex items-center justify-center cursor-pointer hover:scale-105 bg-white',
+        { 'border-black/50': isDisabled }
+      )}
+      data-testid="move-right"
+      on:click={() => handleMoveClick(RIGHT)}
+      >
+      <i
+        class="icon-arrow-up"
+      />
+    </button>
+    <button
+    class={cx('-rotate-90 border border-black h-8 w-8 flex items-center justify-center cursor-pointer hover:scale-105 bg-white',
+        { 'border-black/50': isDisabled }
+      )}
+      on:click={() => handleMoveClick(LEFT)}
+      data-testid="move-left"
+    >
+      <i
+        class="icon-arrow-up"
+      />
+    </button>
+  </div>
+  <div class="w-full flex flex-col gap-2 self-stretch" style={`height: ${height};`}>
+    <span class="text-xs">{ rightlabel }</span>
+    <div class="border border-[#D7D7D] grow px-4 py-3 bg-white flex flex-col overflow-auto">  
+      {#each options.right as option}
+      <button
+        class={cx('flex items-start', { 'bg-[#E2F1FD]': option.selected })}
+        on:click={() => handleOptionClick(option, RIGHT)}
```

Comment:
Double check with @peterborges and @agreenb , but the last I heard about the finalized design decision for this element, it was for the selection to have both the check boxes and the highlight. Where the highlight would be on click, but not select and checking the checkbox would actually select the item to the moveable to the other side